packages/flutter_test/lib/src/window.dart

```diff
@@ -373,4 +373,12 @@ class TestWindow implements Window {
     clearTextScaleFactorTestValue();
     clearViewInsetsTestValue();
   }
+
+  /// This gives us some grace time when the dart:ui side adds something to
+  /// Window, and makes things easier when we do rolls to give us time to
+  /// catch up.
+  @override
+  dynamic noSuchMethod(Invocation invocation) {
```

Comment:
Seems like we should be doing more than just making this a no-op, right? Any time an end-user hits this, it is a legitimate and significant error...

Reply:
This turns the class effectively into a mock.

What are you thinking an end user would do that causes this?  this is meant to make it so that when we add properties to window, we don't break this implementation (requiring manual rolls and mismatched google3/fuchsia contention).

Reply:
If a developer invokes a non-existent method on Window in his/her code, will it throw an exception? Or will it fizzle until some other area of the code fails?

Reply:
It would happen if they were writing a test against TestWindow, casted that to dynamic, and then called a non-existent method/property.

Reply:
This is effectively turning `TestWindow` into a `Mock` instead of a real implementation, for the sake of making rolls less painful.

Reply:
We spoke offline. I thought this was Window, but its TestWindow.